api/policies/hasValidMoveBody.js

```diff
@@ -51,7 +51,9 @@ module.exports = function (req, res, next) {
       }
       return next();
 
-    case MoveType.SCUTTLE:
+      case MoveType.JACK:
+      case MoveType.SCUTTLE:
+      case MoveType.SEVEN_SCUTTLE:
```

Comment:
This is correct, but:
* it looks like the indentation should be decreased one level
* we should delete the below `case MoveType.JACK` from lines 91-108 now that it's being handled here